micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/Home/UserProfile.js

```
@@ -646,39 +875,35 @@ const UserProfile = ({ stateCode, userType, cityDetails }) => {
+                  style={{
+                    marginTop: "24px",
+                    backgroundColor: "#F47738",
+                    width: "100%",
+                    height: "40px",
+                    color: "white",
+                    maxWidth: isMobile ? "100%" : "240px",
+                    borderBottom: "1px solid black",
+                    fontWeight: "700",
+                    fontSize: "17px",
+                  }}
+                >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)